core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -1034,6 +1040,11 @@ private static void validateHideInaccesibleColumns(boolean value, boolean defaul
         }
     }
 
+    public static int getMaxDriversPerQuery(Session session)
```

Comment:
You should only be able to reduce `max_drivers_per_query` and not increase it vs value in `FeatureConfig`